### PR TITLE
Some glaring cross browser issue fixes

### PR DESCRIPTION
**Pull request by stuartsan**
- there was a critical flexbox layout issue in firefox
- basically nothing worked in ie because it needed a couple polyfills
- there were a couple critical flexbox layout issues in ie

TODO

- there is still one glaring issue in ie, you guessed it, having to do with flexbox:

image

possibly will add that on to this PR or might split out depending on timing.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,12 @@
 {
-  "presets": ["env","es2015","stage-0"]
+  "presets": [
+    ["env", {
```
Comment (stuartsan) [inline .babelrc:3]: This is like an automated way to say "please supply whatever polyfills are needed for ie 11 to not explode, given our config".

File: src/initialize.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import 'styles/index';
+import 'babel-polyfill';
```
Comment (stuartsan) [inline src/initialize.js:2]: This works in conjunction with the babel-preset-env plugin -- it is not really importing the entire babel-polyfill collection, just those needed.

File: src/components/EventList.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ export default function(store){
       },
       isSelectedZipcodeInvalid() {
         return this.filters.zipcode &&
-          this.zipcodes &&
```
Comment (cbcowans) [inline src/components/EventList.js:32]: any reason to take the `this.zipcodes` check out?

**Comment by cbcowans** [comment]
`flex` in css is not something I'm super experienced with but it all makes sense to me! 👍

File: src/components/EventList.js (hunk 3 above)
Comment (shazow) [reply src/components/EventList.js:32]: Looks like zipcodes is either an empty object or a filled object, either way the following `Object.keys(this.zipcodes).length` will catch it.

**Comment by shazow** [comment]
:+1: also

**Comment by stuartsan** [comment]
Doin it! Thanks

**Pull request merged**